### PR TITLE
docs (protocol + linting) linting, sectional improvements, and concepts

### PR DESCRIPTION
**Pull request by masterfung**
Improved the concepts sections of protocol. The PR holds linting improvements and removal of certain docs that are dups.

### CODE REVIEW

File: docs/protocol/bugs.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Bugs
```
Comment (fedekunze) [inline docs/protocol/bugs.md:1]: add sidebar position

File: docs/protocol/bugs.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
 
+When creating a Layer 1 (L1) protocol chain, taking security seriously is of utmost importance. L1 protocols are the backbone of the blockchain ecosystem, and any security vulnerabilities within the L1 can lead to a range of potential consequences, from financial loss to reputation damage to even the collapse of the entire network. As such, security is an essential aspect of developing any L1 protocol chain, and security must be considered at every step of the development process. We did conduct an audit with [Informal System](https://github.com/informalsystems/audits/blob/main/Evmos2021Q4/informal-evmos-report-2021q4.pdf).
```
Comment (fedekunze) [inline docs/protocol/bugs.md:3]: I'd link to the audits page instead of linking to a single specific audit

File: docs/protocol/bugs.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
 
-## Bugs Bounty
+Despite our best efforts, security issues may still arise in our L1 protocol chain. In such cases, we encourage our users to report any vulnerabilities or bugs they may find. For sensitive bugs, we ask that users submit the issue to [Immunefi](https://immunefi.com/bounty/evmos/), which is a third-party security platform that helps us identify, triage, and resolve security issues in a confidential and secure manner. For non-sensitive bugs, we encourage users to file an open ticket on our [Evmos Github Repo](https://github.com/evmos/evmos) so that our team can address the issue as quickly as possible.
```
Comment (fedekunze) [inline docs/protocol/bugs.md:5]: nit, split lines

**Review by fedekunze — APPROVED** [review]
LGTM

File: docs/develop/mainnet/api.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Mainnet API
```
Comment (danburck) [inline docs/develop/mainnet/api.md:1]: @masterfung I removed this page on a previous PR and moved the content to https://main--dancing-hummingbird-242a98.netlify.app/develop/mainnet. 

So I'd keep docs/develop/mainnet/api.md removed

File: docs/develop/testnet/api.md [hunk 5]
```diff
@@ -1,3 +1,3 @@
 # Testnet API
 
-We hosted our testnet API endpoints [here](https://api.evmos.dev) and it uses Swagger.
+We hosted our testnet API endpoints [here](https://api.evmos.dev) and it uses Swagger.
```
Comment (danburck) [inline docs/develop/testnet/api.md:3]: Ditto, not sure if we need a whole extra page for this. Instead this content could move to an overview page => docs/develop/testnet

**Review by danburck — APPROVED** [review]
Left some nit comments, otherwise LGTM

File: docs/protocol/bugs.md (hunk 2 above)
Comment (masterfung) [reply docs/protocol/bugs.md:3]: Okay

File: docs/protocol/bugs.md (hunk 1 above)
Comment (masterfung) [reply docs/protocol/bugs.md:1]: Sounds good

File: docs/develop/mainnet/api.md (hunk 4 above)
Comment (masterfung) [reply docs/develop/mainnet/api.md:1]: Sounds good

File: docs/develop/testnet/api.md (hunk 5 above)
Comment (masterfung) [reply docs/develop/testnet/api.md:3]: Will address this in another PR since a ticket was created for this.

**Pull request merged**